ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+Translate.swift

```diff
@@ -86,7 +86,6 @@ extension BrowserViewController: BraveTranslateScriptHandlerDelegate {
         action: { popover in
           popover.previewForOrigin = nil
           popover.dismissPopover()
-          completion(nil)
```

Comment:
This fixes the `CheckedContinuation` crash. `dismissPopover` already calls the `completion`